transmute-ipfs/values.yaml

```diff
@@ -3,7 +3,7 @@ replicaCount: 1
 image: or13/transmute-ipfs
 
 service:
-  type: ClusterIP
+  type: NodePort
```

Comment:
I just changed the type of the ipfs service instead of creating a new one.

Reply:
this makes a lot more sense.